website/content/installation/_index.md

```diff
@@ -61,6 +61,7 @@ To install MetalLB, apply the manifest:
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/namespace.yaml
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/metallb.yaml
+kubectl delete --ignore-not-found=true -f https://raw.githubusercontent.com/metallb/metallb/main/manifests/delete.yaml
```

Comment:
Should also be in release notes, so it's highly visible to people upgrading.

Reply:
removed, replaced with a 1 liner in the release notes